assemblies/features/base/src/main/resources/resources/bin/karaf

```diff
@@ -105,7 +105,7 @@ unlimitFD() {
 
     # Increase the maximum file descriptors if we can
     if [ "${os400}" = "false" ] && [ "${cygwin}" = "false" ]; then
-        if [ "${MAX_FD}" = "maximum" -o "${MAX_FD}" = "max" ]; then
+        if [ "${MAX_FD}" = "maximum" ] || [ "${MAX_FD}" = "max" ]; then
```

Comment:
This line is not necessary an issue but I think it makes sense to change this anyway. For details [have a look at this](https://github.com/koalaman/shellcheck/wiki/SC2166)